### DISCUSSION TITLE
Areabrick / Renderlet not working

### DISCUSSION THREAD

**Discussion by amasotti** [pimcore/pimcore#9249] [Q&A / Support] [answered]
Hi everyone,
I'm watching the first video in Pimcore Academy - Senior Dev Course and trying to code along.
I'm trying to implement a custom Renderlet. I've created an AreaBrick, the associated Twig template and added a method in the default controller. 
When I go to the backend I see my Brick in the Tools > Bundle & Brick list but I'm not able to add it from the (+) button in the edit mode (it just doesn't show in the droplist)
any idea what I'm doing wrong?

**Comment by fashxp** [comment, marked as answer]
Hi, 
did you add it to the `allowed` list of your areablock? 
https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Editables/Areablock/index.html

BR

**Reply by amasotti** [reply]
Now it worked, thanks!
